src/ucp/wireup/select.c

```diff
@@ -982,16 +1052,16 @@ ucp_wireup_add_rma_lanes(const ucp_wireup_select_params_t *select_params,
     ucp_wireup_criteria_init(&criteria);
     if (ep_init_flags & UCP_EP_INIT_FLAG_MEM_TYPE) {
         criteria.title              = "copy across memory types";
-        criteria.local_iface_flags  = UCT_IFACE_FLAG_PUT_SHORT;
-        criteria.remote_iface_flags = UCP_ADDR_IFACE_FLAG_PUT;
+        criteria.local_iface_flags.mandatory  = UCT_IFACE_FLAG_PUT_SHORT;
+        criteria.remote_iface_flags.mandatory = UCP_ADDR_IFACE_FLAG_PUT;
     } else {
         criteria.title              = "remote %s memory access";
-        criteria.remote_iface_flags = UCP_ADDR_IFACE_FLAG_PUT |
-                                      UCP_ADDR_IFACE_FLAG_GET;
-        criteria.local_iface_flags  = UCT_IFACE_FLAG_PUT_SHORT |
-                                      UCT_IFACE_FLAG_PUT_BCOPY |
-                                      UCT_IFACE_FLAG_GET_BCOPY |
-                                      UCT_IFACE_FLAG_PENDING;
+        criteria.remote_iface_flags.mandatory = UCP_ADDR_IFACE_FLAG_PUT |
```

Comment:
maybe use ucp_wireup_init_select_flags here
otherwise align = on column
(in multiple places)